src/components/Dashboard/TodoList.vue

```diff
@@ -539,7 +551,7 @@
           <!-- is this a draft Amalgamation or IA or Registration? -->
```

Comment:
The comment here needs updating. Also, I was thinking if we should move the if statement here to a getter since it might keep growing and growing. Thoughts?

Reply:
Yes. Already working on it :)

Reply:
Thanks! Resolving.